src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -84,6 +86,20 @@ public function register_routes() {
 		);
 	}
 
+	/**
+	 * Sanitize the global styles ID or stylesheet to decode endpoint.
+	 * For example, `wp/v2/global-styles/templatetwentytwo%200.4.0`
+	 * would be decoded to `templatetwentytwo 0.4.0`.
+	 *
+	 * @since 5.9.0
+	 *
+	 * @param string $id_or_stylesheet Global styles ID or stylesheet.
+	 * @param string Sanitized global styles ID or stylesheet.
```

Comment:
We could do with a return value here in the comments.

Reply:
```suggestion
	 * @return string Sanitized global styles ID or stylesheet.
```

Whoopsie, good catch!